test/string_util_gtest.cc

```diff
@@ -160,4 +161,80 @@ TEST(StringUtilTest, StrSplit) {
             std::vector<std::string>({"hello", "there", "is", "more"}));
 }
 
+TEST(StringUtilTest, ExponentToPrefix) {
+  EXPECT_EQ("", benchmark::ExponentToPrefix(0, true));
+  EXPECT_EQ("", benchmark::ExponentToPrefix(0, false));
+
+  EXPECT_EQ("k", benchmark::ExponentToPrefix(1, true));
+  EXPECT_EQ("K", benchmark::ExponentToPrefix(1, false));
```

Comment:
shouldn't this be Ki?

Reply:
Yes. My mistake I'll make the appropriate changes as soon as power resort in my area.